Mixpanel/MixpanelPrivate.h

```diff
@@ -58,6 +58,11 @@
 @property (nonatomic, getter=isValidationEnabled) BOOL validationEnabled;
 #endif
 
+#if !defined(MIXPANEL_WATCH_EXTENSION)
```

Comment:
not sure if this one is possible. UIBackgroundTaskIdentifier and UIViewController don't exist in watchOS